src/components/common/PaginationComponent.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <nav class="flex items-center justify-between border-t border-gray-200 px-4 sm:px-0">
+        <div class="-mt-px flex w-0 flex-1">
+            <button
+                id="pagination-prev-button"
+                @click="coursesStore.setPageNumber(coursesStore.pageNumber - 1)"
+                class="inline-flex items-center border-t-2 border-transparent pt-4 pr-1 text-sm font-medium text-gray-500 hover:border-gray-300 hover:text-gray-700">
+                <ArrowLongLeftIcon class="mr-3 h-5 w-5 text-gray-400" aria-hidden="true" />
+                Previous
+            </button>
+        </div>
+        <div class="hidden md:-mt-px md:flex">
+            <button
+                v-for="button in paginationButtons"
+                :key="button"
+                id="pagination-button"
+                @click="coursesStore.setPageNumber(button)"
+                class="inline-flex items-center border-t-2 px-4 pt-4 text-sm font-medium text-gray-500 hover:border-gray-300 hover:text-gray-700"
+                :class="button === coursesStore.pageNumber ? 'border-indigo-500 text-indigo-600' : 'border-transparent'"
+            >{{button}}</button>
```

Comment:
FYI: It may be better to move `{button}` to the new separate line to keep the same style as you have everywhere in the project

Reply:
Dima will able to fix it after lesson, when we will learn prettier and eslint. This problems are fixed by one command like "npm run format"

but, I believe it is a good point to know about the famous code styles